tests/integration/conftest.py

```diff
@@ -23,11 +23,11 @@
 
 
 class Environ(Enum):
-    LOCAL = 'local'
-    PROD = 'prod'
-    STAGING = 'staging'
-    ONPREM = 'onprem'
-    CUSTOM = 'custom'
+    LOCAL = "local"
```

Comment:
How come we're changing these double quotes? It's best practice to use single quotes these, iirc